src/main/java/org/openrewrite/java/dependencies/github/ParseAdvisories.java

```diff
@@ -92,16 +93,20 @@ public FileVisitResult visitFile(Path path, BasicFileAttributes attrs) throws IO
                             && affected.getRanges() != null
                             && !affected.getRanges().isEmpty()) {
                         Range range = affected.getRanges().iterator().next();
+                        String cve = advisory.getAliases().isEmpty() ?
+                                advisory.getId() :
+                                advisory.getAliases().iterator().next();
+                        List<String> cweIds = advisory.getDatabaseSpecific().getCweIds();
+                        String cwes = cweIds == null || cweIds.isEmpty() ? null : String.join(";", cweIds);
```

Comment:
I suggest to use a different separator e.g `|` to avoid confusion with the current one `,`

Reply:
Hmm yeah I went with a semicolon, as that's what Jackson generates when it serializes a list by default, and if this is in turn parsed I'd prefer that to use that default over forcing people to pick an alternative explicitly. It's also slightly more common when presented one to one in a user interface, so I'd opt to keep it this way. Thanks for the detailed review though!